specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -106,41 +106,1303 @@
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/changepoint/detect": {
+      "post": {
+        "summary": "Detect change point for the interval series",
+        "description": "Evaluate change point score of every series point between start and end.",
+        "operationId": "TimeSeries_ChangePointDetectOnTimestamp",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Timestamp is needed. Advanced model parameters can also be set in the request if needed.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectOnTimestampRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectOnTimestampResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Changepoint example": {
+            "$ref": "./examples/ChangePointDetectOnTimestamp.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/label": {
+      "post": {
+        "summary": "Label API is used for users to label detection status of a certain time stamp of a time series",
+        "description": "This operation is used for users to label Anomaly or ChangePoint states of a certain time stamp, these label will be used for regenerate detection model",
+        "operationId": "TimeSeries_Label",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "In Label request, user can set Anomaly|ChangePoint state (true, false, unknown) for a time range",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LabelRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Label example": {
+            "$ref": "./examples/Label.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/query": {
+      "post": {
```

Comment:
consider paging in query API

Reply:
Resolved, no need paging since the API will limit the response within 8640 points. Begin will always be close, end will be open